src/Easydropdown/Easydropdown.ts

```diff
@@ -99,6 +99,16 @@ class Easydropdown {
         Renderer.queryDomRefs(this.dom, ['group', 'option', 'item']);
     }
 
+    public validate(): boolean {
+        if (this.state.selectedIndex !== 0 || this.state.isRequired !== true) {
```

Comment:
This logic will only work if the select has a "placeholder" option, i.e. a first option with an empty value attribute, specifically to be used as an initial placeholder/label.

If there is no placeholder, then the selectedIndex will be `0` by default, but the dropdown will still be valid, because the first option will have a value.

There is a `hasValue` getter in the state that is nicely agnostic to whether a placeholder exists, and simply checks if the value is empty string, so I would recommend the following instead:

```ts
public validate(): boolean {
    if (!this.state.isRequired || this.state.hasValue) {
        return true;
    }

    ...
}
```

Note that I've moved the `isRequired` lookup first as I feel it makes sense to avoid the more expensive getter computation if no validation is required.